modules/util/portal-tools-rest-builder/bnd.bnd

```diff
@@ -7,6 +7,7 @@ Import-Package:\
 	\
 	*
 Main-Class: com.liferay.portal.tools.rest.builder.RESTBuilder
+-fixupmessages: "^Classes found in the wrong directory: \\\\{META-INF/versions/9/";is:=warning
```

Comment:
@jeffwu0724 I do not think you need this.

Reply:
@tinatian Yes you are right, since we exclude it while expanding compileInclude, then we do not have `META-INF/versions/9/` and more. (also checked locally)

I will make a quick fix and force push